iree/tools/android/run_module_app/README.md

```diff
@@ -0,0 +1,62 @@
+# iree-run-module Android App
+
+This directory contains an Android application packing the `iree-run-module`
+command-line tool together with a specific IREE VM module invocation.
+
+Note that this app is **purely** for benchmarking/profiling IREE itself.
+This is not the expected integration path for a real Android application,
+for which we expect to provide proper Java API and build support.
+
+## Native Activity
+
+The app uses Android [`NativeActivity`][native-activity] to bridge IREE core
+libraries together with the Android system. Native activity allows one to
+implement an Android Activity purely in C/C++. There are
+[tutorials][native-activity-tutorial] and [examples][native-activity-example]
```

Comment:
These don't seem like real links?

Reply:
These links are attached at the end of this file. I typically like this because it results in cleaner README.